service/controller/aws/v7/resource/chartconfig/resource.go

```diff
@@ -0,0 +1,57 @@
+package chartconfig
```

Comment:
The chartconfig resource uses the chartconfig service to create the CRs. But lets us inject provider specific charts.